controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2268 @@
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - grub2_enable_selinux
+
+  - id: 1.7.1.3
+    title: Ensure SELinux policy is configured (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - var_selinux_policy_name=targeted
+      - selinux_policytype
+
+  - id: 1.7.1.4
+    title: Ensure the SELinux state is enforcing (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - var_selinux_state=enforcing
+      - selinux_state
+
+  - id: 1.7.1.5
+    title: Ensure no unconfined services exist (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - selinux_confinement_of_daemons
+
+  - id: 1.7.1.6
+    title: Ensure SETroubleshoot is not installed (Automated)
+    levels:
+      - l2_server
+    automated: yes
+    rules:
+      - package_setroubleshoot_removed
+
+  - id: 1.7.1.7
+    title: Ensure the MCS Translation Service (mcstrans) is not installed (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - package_mcstrans_removed
+
+  - id: 1.8.1.1
+    title: Ensure message of the day is configured properly (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - banner_etc_motd
+
+  - id: 1.8.1.2
+    title: Ensure local login warning banner is configured properly (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - banner_etc_issue
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5225
+  - id: 1.8.1.3
+    title: Ensure remote login warning banner is configured properly (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 1.8.1.4
+    title: Ensure permissions on /etc/motd are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_permissions_etc_motd
+
+  - id: 1.8.1.5
+    title: Ensure permissions on /etc/issue are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_permissions_etc_issue
+
+  - id: 1.8.2
+    title: Ensure GDM login banner is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - dconf_gnome_banner_enabled
+      - dconf_gnome_login_banner_text
+
+  - id: 1.9
+    title: Ensure updates, patches, and additional security software are installed (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    related_rules:
+      - security_patches_up_to_date
+
+  - id: 1.10
+    title: Ensure system-wide crypto policy is not legacy (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - configure_crypto_policy
```

Comment:
The rule is missing a cis reference.
Additionally, I suggest we add

```
      - var_system_crypto_policy=default
```

Reply:
I'm not sure I can find where this rule is missing the CIS reference. It seems to have one for RHEL 8 but not RHEL 7:
https://github.com/ComplianceAsCode/content/blob/2b2152d288e05f0d64f26fff3f01b0e75311023d/linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/rule.yml#L64

Additionally, if we set `var_system_crypto_policy` to `default` here, how does this interact with what I'm doing for CIS 1.11 below? If two variables are set in a controls file, which one ends up taking precedence?

Reply:
You are right, I overlooked it cause the rule has actually two references. And they are correct.